sig-scalability/README.md

```diff
@@ -13,7 +13,7 @@ SIG Scalability is responsible for defining and driving scalability goals for Ku
 The [charter](charter.md) defines the scope and governance of the Scalability Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [Thursdays at 18:30 Warsaw](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (bi-weekly ([upcoming meeting dates](#upcoming-2019-meeting-dates))). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:30&tz=Warsaw).
+* Regular SIG Meeting: [Thursdays (bi-weekly) at 17:30 UTC](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) ([upcoming meeting dates](#upcoming-2020-meeting-dates)). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
```

Comment:
This was causing issues in the past, I think...

Reply:
I thought time in UTC is the invariant, no?

Reply:
I can't remember what the problem was exactly, but it used to be UTC in the past (unless I completely messed something up).

Reply:
I've looked in the timezone converter http://www.thetimezoneconverter.com/?t=17:30&tz=UTC and it seems to convert fine to Warsaw and Pacific time.

I'm not sure though if will convert fine event after DST ends (don't know how to verify it there).

Reply:
Yeah, it was UTC and we changed it to Warsaw time, but I don't really remember what the logic behind it was :)
But I see your point, UTC is not affected by DST, so there will be less changes for countries that use DST. E.g. currently time in US changes (so the meeting time for US people changes) and then after a week or so time in Poland changes, and the meeting time changes again for people in US. With UTC there will be only one change per DST start/end.

Reply:
Ok, then it seems fine for now. Let's fix if it breaks after DST ends.